libcontainer/utils/utils.go

```diff
@@ -19,6 +19,9 @@ const (
 	exitSignalOffset = 128
 )
 
+// RunInUserns is the user namespace state of the process.
+var RunInUserns = -1
```

Comment:
You can use this https://github.com/opencontainers/runc/blob/e7461c8c29ab387962255e12ea50d02fc29bb4ec/libcontainer/userns/userns.go#L5

Reply:
Any why is this in mountPropagate()? Doesn’t seem relevant to mount propagation

Reply:
> You can use this
> 
> https://github.com/opencontainers/runc/blob/e7461c8c29ab387962255e12ea50d02fc29bb4ec/libcontainer/userns/userns.go#L5

Oh, thanks, i just know that there is `RunningInUserNS` in the code

> Any why is this in mountPropagate()? Doesn’t seem relevant to mount propagation

In fact, this logic should be placed in `mountToRootfs`, but this will make it a separate mount operation, so I will put it in `mountPropagate`.

If it is not suitable, it can still be put into `mountToRootfs`